website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,288 @@
+# Operator Scope
+
+## Overview
+
+A `namespace-scoped` operator watches and manages resources in a single namespace, whereas a `cluster-scoped` operator watches and manages resources cluster-wide. 
+
+A `cluster-scoped` scope is ideal for operator projects which will control resources in more than one namespace. At the same time, `namespace-scoped` operators are preferred because of their flexibility. They enable decoupled upgrades, namespace isolation for failures and monitoring, and differing API definitions.
+
+However, there are use cases where a `cluster-scoped` operator may make sense. For example, the [cert-manager][cert-manager] operator is often deployed with cluster-scoped permissions and watches so that it can manage to issue certificates for an entire cluster.
```

Comment:
This sentence doesn't really follow well from the previous paragraph. Something along the lines of the following would be better:
```
By default, `operator-sdk init` scaffolds a cluster-scoped operator.
This document details conversion of default operator projects to namespaced-scoped operators.
```